__tests__/app.test.js

```diff
@@ -190,4 +190,82 @@ describe('Testing app', () => {
           .expect(204);
         });
     });
+    describe('POST', () => {
+        test('status:201, responds with a new comment added to an article', () => {
+          const testComment = {
+            article_id: 1,
+            username: 'butter_bridge',
+            body: `I can't read so the article was lost on me. Needs more pictures!`,
+          };
+          return request(app)
+            .post('/api/articles/1/comments')
+            .send(testComment)
+            .expect(201)
+            .then(({ body }) => {
+                expect(body.comment).toEqual(
+                    expect.objectContaining({
+                        article_id: 1,
+                        author: 'butter_bridge',
+                        body: `I can't read so the article was lost on me. Needs more pictures!`,
+                    })
+                )
+            });
+        });
+        test('should return a status error of 400 if there is missing info in the body ', () => {
+            const badComment = {
+              article_id: 1,
+              username: 'icellusedkars',
+              body: '',
+            }
+            return request(app)
+            .post("/api/articles/1/comments")
+            .send(badComment)
+            .expect(400)
+            .then(({ body: { msg } }) => {
+              expect(msg).toBe('Missing Input!')
+            })
+        })
+        test('should return a status error of 400 if invalid info is entered', () => {
+            const anotherBadComment = {
+                article_id: 1,
+                username: 3,
+                body: 'Bob Lob Law',
+              }
+            return request(app)
+            .post("/api/articles/1/comments")
+            .send(anotherBadComment)
+            .expect(400)
+            .then(({ body: { msg } }) => {
+              expect(msg).toBe('Incorrect Input!')
+            })
+        });
+        test('should return a status error of 400 if invalid article id is entered', () => {
+            const wrongArticleComment = {
+                article_id: 99999,

```

Comment:
Same for these -  you can just put the article_id in the route, and it should be handled from there